core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -3120,6 +3120,19 @@ public String toString() {
               + "`alluxio.client.file.cache.LRUCacheEvictor`.")
           .setScope(Scope.CLIENT)
           .build();
+  public static final PropertyKey USER_CLIENT_CACHE_DIR =
+      new Builder(Name.USER_CLIENT_CACHE_DIR)
+          .setDefaultValue("/tmp")
+          .setDescription("The directory where client-side cache is stored.")
+          .setScope(Scope.CLIENT)
+          .build();
+  public static final PropertyKey USER_CLIENT_CACHE_STORE_TYPE =
+      new Builder(Name.USER_CLIENT_CACHE_STORE_TYPE)
+          .setDefaultValue("LOCAL")
+          .setDescription("The type of page store to use for client-side cache. Can be either "
+              + "`LOCAL` or `ROCKS`.")
```

Comment:
specify a bit more here what is `LOCAL` and what is `ROCKS`?

Reply:
Done.